app/plugins/system_controller/system/index.js

```diff
@@ -1045,4 +1046,52 @@ ControllerSystem.prototype.getCPUCoresNumber = function () {
       self.logger.error('Could not retrieve CPU Cores: ' + e);
       return 1
     }
-};
\ No newline at end of file
+};
+
+ControllerSystem.prototype.enableLiveLog = function (data) {
+  if (data === 'true') {
+    try {
+      this.logger.info('Launching a new LiveLog session');
+      const format = 'cat'; // json is also an option for more serious logging/filtering
+      const args = ['--output', format, '-f'];
+      const defaults = {
+        cwd: undefined,
+        env: process.env
+      };
+      const liveLogData = {
+        message: 'Starting Live Log...\n'
+      };
+      this.commandRouter.broadcastMessage('LLogOpen', liveLogData);
+      
+      if (this.livelogchild) {
+        this.livelogchild.kill();
+      }
+      this.livelogchild = spawn('journalctl', args, defaults); // sudo or not?
```

Comment:
Should not require sudo, but we have an issue that in system updated from older installs the users file does not get updated properly. 
Another good practice is to use the full path.
I would do: /usr/bin/sudo /bin/journalctl -f

Reply:
fixed!